src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -78,19 +78,22 @@ public static void setDatesToRepoConfigs(
      */
     public static void merge(List<RepoConfiguration> repoConfigs, List<RepoConfiguration> authorConfigs) {
         for (RepoConfiguration authorConfig : authorConfigs) {
-            int index = repoConfigs.indexOf(authorConfig);
-
-            if (index == -1) {
-                logger.warning(String.format(
-                        "Repository %s is not found in repo-config.csv.", authorConfig.getLocation()));
-                continue;
+            if (authorConfig.location.isEmpty()) {
+                for (RepoConfiguration repoConfig : repoConfigs) {
+                    repoConfig.addAuthors(authorConfig.getAuthorList());
+                }
+            } else {
+                int index = repoConfigs.indexOf(authorConfig);
+
+                if (index == -1) {
+                    logger.warning(String.format(
+                            "Repository %s is not found in repo-config.csv.", authorConfig.getLocation()));
+                    continue;
+                }
+
+                RepoConfiguration repoConfigToAdd = repoConfigs.get(index);
```

Comment:
`repoConfigToAdd` is confusing. 

Keep it simple, `repoConfigToAdd` -> `repoConfig`?